test/runtests.jl

```diff
@@ -34,8 +36,7 @@ for pardiso_type in psolvers
         X = similar(B)
 
         # Test unsymmetric, herm indef, herm posdef and symmetric
-        for A in SparseMatrixCSC[A1, A1 + A1', A1'A1, A1 + A1.']
-
+        for A in SparseMatrixCSC[A1, A1 + A1', A1'A1, copy(transpose(A1 + A1))]
```

Comment:
Should this be `copy(A1 + transpose(A1))`  instead of `copy(transpose(A1 + A1))` ?

Reply:
Indeed, thanks.